src/amber/cli/commands/pipelines.cr

```diff
@@ -0,0 +1,92 @@
+require "cli"
+require "shell-table"
+require "../helpers/sentry"
+
+module Amber::CLI
+  class MainCommand < ::Cli::Supercommand
+    class Pipelines < Command
+      # TODO: use hash with fixed order?
```

Comment:
It would be nice to print the pipes in the same order they are used and maybe somehow reflect that in the output

Reply:
Agreed, they should display in order - maybe that is already happening. The test should reflect that.